backend/capellacollab/sessions/operators/__init__.py

```diff
@@ -2,15 +2,15 @@
 # SPDX-License-Identifier: Apache-2.0
 
 
-from capellacollab.sessions.operators.k8s import KubernetesOperator
+import functools
 
-operator = None
+from capellacollab.sessions.operators.k8s import KubernetesOperator
 
 
 def load_operator():
-    global operator
-    operator = KubernetesOperator()
+    get_operator()
```

Comment:
We can get rid of the `load_operation` function. Just call `get_operator` instead.

Reply:
Removed the `load_operator` and added a comment to `__main__` that the `get_operator` is needed there to load the kubernetes config at startup